pkg/client/cli/cmd_loglevel.go

```diff
@@ -0,0 +1,96 @@
+package cli
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"google.golang.org/protobuf/types/known/durationpb"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/telepresenceio/telepresence/rpc/v2/connector"
+	"github.com/telepresenceio/telepresence/rpc/v2/daemon"
+	"github.com/telepresenceio/telepresence/rpc/v2/manager"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/cliutil"
+)
+
+const defaultDuration = 30 * time.Minute
+
+type logLevelSetter struct {
+	duration   time.Duration
+	localOnly  bool
+	remoteOnly bool
+}
+
+func logLevelArg(cmd *cobra.Command, args []string) error {
+	if len(args) != 1 {
+		return errors.New("accepts exactly one argument (the log level)")
+	}
+	lvl, err := logrus.ParseLevel(args[0])
+	if err != nil {
+		return err
+	}
+	switch lvl {
+	case logrus.PanicLevel, logrus.FatalLevel:
+		return fmt.Errorf("unsupported log level: %s", lvl)
+	}
+	return nil
+}
+
+func loglevelCommand() *cobra.Command {
+	lvs := logrus.AllLevels[2:] // Don't include `panic` and `fatal`
+	lvStrs := make([]string, len(lvs))
+	for i, lv := range lvs {
+		lvStrs[i] = lv.String()
+	}
+	lls := logLevelSetter{}
+	cmd := &cobra.Command{
+		Use:       fmt.Sprintf("loglevel <%s>", strings.Join(lvStrs, ",")),
+		Args:      logLevelArg,
+		Short:     "Temporarily change the log-level of the traffic-manager and the traffic-agents",
```

Comment:
maybe do "traffic-manager, traffic-agent, and client processes" since this can also be used to set the user + root daemons